web/codechecker_web/shared/util.py

```diff
@@ -0,0 +1,390 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+Util module.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import datetime
+import hashlib
+import json
+import os
+import re
+import shutil
+import socket
+import stat
+import subprocess
+import tempfile
+import uuid
+
+
+from codechecker_common.logger import get_logger
+
+LOG = get_logger('system')
+
+
+class CmdLineOutputEncoder(json.JSONEncoder):
```

Comment:
I have created a `codechecker_client/cmd_line.py` because `codechecker_client/cmd` package is already exists and it will be loaded instead of `codechecker_client/cmd.py` module.